global/scripts/parallel.shlib

```diff
@@ -172,3 +185,75 @@ function par_addjob_stdin()
     par_jobinputarray+=("$jobinput")
 }
 
+#Add job to the current stage (FSLSUB, BUILTIN or NONE mode)
+#Inputs:
+#1. parallel mode
+#2. prallel job parameter (queue name for FSLSUB, must be 'NONE' for others) 
+#3. command with arguments
+function par_add_job_to_stage()
+{
+    local PARALLEL_MODE=$1    	
+    echo "parallel mode: $PARALLEL_MODE"
```

Comment:
This looks like it will be very uninformative to see in the log/terminal after every job submission.

Reply:
ok